packages/core/src/js/feedback/FeedbackForm.types.ts

```diff
@@ -8,6 +11,11 @@ export interface FeedbackFormProps extends FeedbackGeneralConfiguration, Feedbac
  * General feedback configuration
  */
 export interface FeedbackGeneralConfiguration {
+  /**
+   * Show the Sentry branding
```

Comment:
We should show the default value so users know what to expect if they don't change it.

```suggestion
   * Show the Sentry branding
   *
   * @default true
```

Reply:
Good idea 👍